tools/dependency/validate.py

```diff
@@ -60,7 +60,7 @@ def TestOnlyIgnore(dep):
   if dep.startswith('remotejdk'):
     return True
   # Python (pip3)
-  if '_pip3_' in dep:
+  if '_pip3' in dep:
```

Comment:
@htuch im not entirely clear if this is the correct solution - im guessing using the old rules_proto the targets were registered differently and so this is correct

im not totally sure if it would now exclude some targets that *should* be validated tho

Reply:
This is fine, we're not validating Python here.